packages/vite-plugin/src/index.ts

```diff
@@ -56,15 +58,17 @@ export function vanillaExtractPlugin({ identifiers }: Options = {}): Plugin {
       packageInfo = getPackageInfo(config.root);
     },
     resolveId(id) {
-      if (id.indexOf(virtualExt) > 0) {
+      if (id.startsWith(virtualPrefix) && id.indexOf(virtualExt) > 0) {
         return id;
       }
     },
     load(id) {
       const extensionIndex = id.indexOf(virtualExt);
 
-      if (extensionIndex > 0) {
-        const fileScopeId = id.substring(0, extensionIndex);
+      if (id.startsWith(virtualPrefix) && extensionIndex > 0) {
```

Comment:
Think we can drop the `extensionIndex` check here as well as it's now redundant. This means we can pull the `extensionIndex` variable into the if block scope as well.

Reply:
Done these, too.